x-pack/plugin/otel-data/src/main/resources/component-templates/otel@mappings.yaml

```diff
@@ -0,0 +1,65 @@
+---
+version: ${xpack.oteldata.template.version}
+_meta:
+  description: Default mappings for all OTel data streams
+  managed: true
+template:
+  mappings:
+    date_detection: false
+    dynamic: false
+    properties:
+      "@timestamp":
+        type: date
+      data_stream.type:
+        type: constant_keyword
+      data_stream.dataset:
+        type: constant_keyword
+      data_stream.namespace:
+        type: constant_keyword
+      attributes:
+        type: passthrough
+        dynamic: true
+        priority: 10
+        time_series_dimension: true
+      dropped_attributes_count:
+        type: long
+      processor.event:
```

Comment:
Remove `processor.event` to allow attributes.processor.event

Reply:
Done in https://github.com/elastic/elasticsearch/pull/111091/commits/5b4b65f71825eb40ab84fd3caead07506c74d409